src/view/menu.py

```diff
@@ -89,4 +107,4 @@ def _process_events(self, events: List[EventType]) -> bool:
                     self.selected_item = (self.selected_item + 1) % len(self._options)
                 elif event.key == pygame.K_RETURN or event.key == pygame.K_KP_ENTER:
                     return True
-        return False
+        return False
```

Comment:
No End Of Line sign at the end of the file
```suggestion
        return False

```